q2_demux/test/test_transformer.py

```diff
@@ -56,7 +56,7 @@ def test_emp_se_multiplexed_format_barcode_sequence_iterator(self):
         transformer1 = self.get_transformer(EMPMultiplexedSingleEndDirFmt,
                                             EMPMultiplexedDirFmt)
         transformer2 = self.get_transformer(EMPMultiplexedDirFmt,
-                                            BarcodeSequenceIterator)
+                                            BarcodeSequenceFastqIterator)
         dirname = 'emp_multiplexed_single_end'
         dirpath = self.get_data_path(dirname)
         emp_demultiplexed = \
```

Comment:
There's tests with commented out code, is that ready to be uncommented in this PR? 🎱